platforms/gke/base/core/container_cluster/container_cluster.tf

```diff
@@ -200,33 +171,13 @@ resource "google_container_cluster" "cluster" {
 
   node_pool {
     initial_node_count = 1
-    name               = "system"
-
-    autoscaling {
-      location_policy      = "BALANCED"
-      total_max_node_count = 32
-      total_min_node_count = 1
-    }
-
-    network_config {
-      enable_private_nodes = true
-    }
-
+    name               = "default-pool"
```

Comment:
Is it possible to have a variable here? It will be useful for the federated learning use case.

Reply:
This has to be `default-pool`, it gets deleted because `remove_default_node_pool = true`

Reply:
Ah got it. Didn't notice the `remove_default_node_pool` bit. We can close this.